docs/xdr/features/collect/integrations/endpoint/sekoiaio.md

```diff
@@ -117,40 +120,86 @@ The Endpoint Detection Agent is easy to install on Windows or Linux systems once
     sudo systemctl status SEKOIAEndpointAgent.service
     ```
 
-Once installed, the agent collects, normalizes, and sends event logs to Sekoia.io. The protocol used to send events is HTTPS (443).
-
 #### journald configuration
```

Comment:
Journald should be under the linux tab. (We also have the issue in the currently deployed documentation)